phalcon/application.zep

```diff
@@ -0,0 +1,150 @@
+
+/*
+ +------------------------------------------------------------------------+
+ | Phalcon Framework                                                      |
+ +------------------------------------------------------------------------+
+ | Copyright (c) 2011-2015 Phalcon Team (http://www.phalconphp.com)       |
+ +------------------------------------------------------------------------+
+ | This source file is subject to the New BSD License that is bundled     |
+ | with this package in the file docs/LICENSE.txt.                        |
+ |                                                                        |
+ | If you did not receive a copy of the license and are unable to         |
+ | obtain it through the world-wide-web, please send an email             |
+ | to license@phalconphp.com so we can send you a copy immediately.       |
+ +------------------------------------------------------------------------+
+ | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
+ |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ +------------------------------------------------------------------------+
+ */
+
+namespace Phalcon;
+
+use Phalcon\Application\Exception;
+use Phalcon\DiInterface;
+use Phalcon\Di\Injectable;
+use Phalcon\Events\EventsAwareInterface;
+use Phalcon\Events\ManagerInterface;
+
+/**
+ * Phalcon\Application
+ *
+ * Base class for Phalcon\Cli\Console and Phalcon\Mvc\Application.
+ */
+abstract class Application extends Injectable implements EventsAwareInterface
+{
+
+	protected _eventsManager;
+
+	protected _dependencyInjector;
+
+	protected _defaultModule;
+
+	protected _modules;
+
+	/**
+	 * Phalcon\Application
+	 */
+	public function __construct(<DiInterface> dependencyInjector = null)
+	{
+		if typeof dependencyInjector == "object" {
+			let this->_dependencyInjector = dependencyInjector;
+		}
+	}
+
+	/**
+	 * Sets the events manager
+	 */
+	public function setEventsManager(<ManagerInterface> eventsManager)
+	{
+		let this->_eventsManager = eventsManager;
+	}
+
+	/**
+	 * Returns the internal event manager
+	 */
+	public function getEventsManager() -> <ManagerInterface>
+	{
+		return this->_eventsManager;
+	}
+
+	/**
+	 * Register an array of modules present in the application
+	 *
+	 *<code>
+	 *	$this->registerModules(array(
+	 *		'frontend' => array(
+	 *			'className' => 'Multiple\Frontend\Module',
+	 *			'path' => '../apps/frontend/Module.php'
+	 *		),
+	 *		'backend' => array(
+	 *			'className' => 'Multiple\Backend\Module',
+	 *			'path' => '../apps/backend/Module.php'
+	 *		)
+	 *	));
+	 *</code>
```

Comment:
can you please remove extra slashes here? also use short array syntax. we are in the 21st century :)